lib/api/build.rs

```diff
@@ -165,7 +165,11 @@ fn configure_validation(builder: Builder) -> Builder {
             ("RecommendPointGroups.limit", "range(min = 1)"),
             ("RecommendPointGroups.params", ""),
             ("CountPoints.collection_name", "length(min = 1, max = 255)"),
+            ("GeoPolygon.points", "custom = \"crate::grpc::validate::validate_geo_polygon\""),
         ], &[])
+        .type_attribute("NamedVectors", "#[derive(serde::Serialize)]")
```

Comment:
Here as well, the `Serialize` seemed to have been resurrected after https://github.com/qdrant/qdrant/pull/2225

Reply:
removed.